packages/launch-config/src/types/constants.ts

```diff
@@ -1,3 +1,5 @@
 export const FIORI_TOOLS_LAUNCH_CONFIG_HANDLER_ID = 'fiori_tools';
 export const LAUNCH_JSON_FILE = 'launch.json';
 export const RUN_SCRIPT = 'run-script';
+export const oDataVersionV2: string = '2',
+    oDataVersionV4: string = '4';
```

Comment:
Please do not create own constants for this but use type
https://github.com/SAP/open-ux-tools/blob/ddd43c393a040c23ffe158e06e7e376426bfe53f/packages/project-access/src/types/info/index.ts#L75